docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -263,11 +263,17 @@ spec:
   storageClassName: ''
   tenants:
     mode: openshift-logging
+  limits:
+    global:
+      ingestion:
+        ingestionBurstSize: 9
+        ingestionRate: 5
 ----
```

Comment:
I'd probably add a note regarding the unit of the parameters, and replicate the math from the PR description in the docs.

```suggestion
  limits:
    global:
      ingestion:
        ingestionBurstSize: 9 <1>
        ingestionRate: 5 <1>
----
<1> values are in MB/s
```